### ISSUE TITLE
Add some gestures for pro users

### ISSUE THREAD

**Issue by a2l0i7m0** [8bitPit/Niagara-Issues#2661]
### Is your feature request related to a problem?

Add some gestures for pro users.  Swipe down, left, right as a gesture

### Describe the solution you'd like

Add some gestures for pro users.  Swipe down, left, right as a gesture

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by TFackler** [comment]
> Customizable gestures are a popular feature in many launchers, but at Niagara Launcher, we believe in keeping things simple and easy to manage. Therefore, we have made some choices about the gestures we support to give you the best user experience.
[...]
We do not allow [...] horizontal swipe gestures (left and right) since they are already assigned to opening pop-ups and dismissing notifications. Similarly, we do not support custom vertical swipe gestures (up and down) since they are reserved for essential functions like opening the notification drawer and the Niagara search.
However, we recommend these alternatives. You can add the apps you frequently use to your favorites and use Niagara Launcher's [pop-ups](https://help.niagaralauncher.app/article/115-pop-ups) for other frequently used apps. Additionally, you can customize the tap gesture and the swipe-up gesture of our [Niagara button](https://help.niagaralauncher.app/article/116-niagara-button) with Niagara Pro.

(from https://help.niagaralauncher.app/article/77-missing-customization-features)

In short, we do not want to add more gestures, since they will complicate usage and most likely make interacting with other UI elements more difficult.
